src/composables/trade/useSor.ts

```diff
@@ -169,9 +168,10 @@ export default function useSor({
     const poolsUrlV1 = `${
       configService.network.poolsUrlV1
     }?timestamp=${Date.now()}`;
-    const poolsUrlV2 = `${
-      configService.network.poolsUrlV2
-    }?timestamp=${Date.now()}`;
+    // TODO: pull a set of SubgraphPoolBase[] data from here
+    // const poolsUrlV2 = `${
```

Comment:
This didn't seem to get used before (as we'd always overwrite with subgraph data). Shall we just get rid of these?

Reply:
Yeah, I think this was leftover from a previous method of getting the data.

Reply:
Cool, I'll get rid of the V2 version